examples/index.html

```diff
@@ -186,7 +186,7 @@ <h2 style="margin-top:20px">Save</h2>
                 <option>Save sequence string</option>
               </select>
               <button id="saveButton" class="btn btn-primary btn-lg" style="margin-top:20px; margin-bottom:0px;align:center; width:100%;">Save</button>
-              <button class="btn btn-default btn-lg js-view-as-gif" id="gif" style="margin: 0px 35px 0px 35px;width:100%;">Preview GIF</button>
+              <button class="btn btn-default btn-lg js-view-as-gif" id="gif" style="margin-left:0px;margin-top:5px;width:100%;">Preview GIF</button>
```

Comment:
This has a class of 
`.js-view-as-gif`. How about putting the styles in demo.css?

Reply:
You can do the same for save btn as well